packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -1251,6 +1251,7 @@ module.exports = function (log, config) {
         passwordManagerInfoUrl: links.passwordManagerInfoUrl,
         privacyUrl: links.privacyUrl,
         resetLink: links.resetLink,
+        supportUrl: links.supportUrl,
```

Comment:
This was actually missing in the current template and the link was rendering as undefined, but I don't think we send this email too often so it probably was easily missed.

Reply:
Great catch!